x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row_health_pane.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { type FC } from 'react';
+
+import { formatDate, EuiSpacer, EuiInMemoryTable } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { TIME_FORMAT } from '../../../../../../common/constants';
+import type { TransformHealthIssue } from '../../../../../../common/types/transform_stats';
+
+import { TransformListRow } from '../../../../common';
+
+import { TransformHealthColoredDot } from './transform_health_colored_dot';
+
+interface ExpandedRowHealthPaneProps {
+  health: TransformListRow['stats']['health'];
+}
+
+export const ExpandedRowHealthPane: FC<ExpandedRowHealthPaneProps> = ({ health }) => {
+  const { status, issues } = health;
+
+  const sorting = {
+    sort: {
+      field: 'first_occurrence' as const,
+      direction: 'desc' as const,
+    },
+  };
+
+  const columns = [
+    {
+      field: 'first_occurrence',
+      name: i18n.translate(
+        'xpack.transform.transformList.transformDetails.healthPane.firstOccurrenceLabel',
+        {
+          defaultMessage: 'First Occurrence',
+        }
+      ),
+      render: (firstOccurrence: number) => formatDate(firstOccurrence, TIME_FORMAT),
+      sortable: true,
+    },
+    {
+      field: 'count',
+      name: i18n.translate('xpack.transform.transformList.transformDetails.healthPane.countLabel', {
+        defaultMessage: 'count',
+      }),
+      sortable: true,
+      width: '60px',
+    },
+    {
+      field: 'issue',
+      name: i18n.translate('xpack.transform.transformList.transformDetails.healthPane.issueLabel', {
+        defaultMessage: 'Issue',
+      }),
+      sortable: true,
+    },
+    {
+      field: 'details',
+      name: i18n.translate(
+        'xpack.transform.transformList.transformDetails.healthPane.detailsLabel',
+        {
+          defaultMessage: 'Details',
+        }
+      ),
+      width: '50%',
+    },
+  ];
+
+  return (
+    <div data-test-subj="transformHealthTabContent">
+      <EuiSpacer size="s" />
+      <TransformHealthColoredDot healthStatus={status} compact={false} />
```

Comment:
Can't decide about the styling of the text which goes with the health dot. Currently it is the same style as used in the cells for the main table row, which makes it hard to distinguish the rows from the expanded row content. Would a smaller font work better for this text?

<img width="861" alt="image" src="https://user-images.githubusercontent.com/7405507/218099025-b7336e71-7a24-4edb-9e87-f050020dc139.png">